### PR TITLE
Server requests oauth token from GitHub with provided code

### PR DESCRIPTION
**Pull request by nakhbari**
- [ ] I have run `rspec` and corrected all errors
- [ ] I have run `rubocop` and corrected all errors
- [ ] I have run `npm run test` and corrected all errors
- [ ] I have tested this change locally and tried to launch the server as well as access a project, and with that at least one build
- [ ] If there is an existing issue, make sure to add `Fixes ...` as part of the PR body to reference the issue you're solving

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -33,6 +33,9 @@ gem "octokit", ">= 4.8.0", "< 5.0.0"
 # Load the `.keys` dotenv file we use to store encryption data
 gem "dotenv", ">= 2.4.0", "< 3.0.0"
 
+## Simplified HTTP client library
+gem "faraday", ">= 0.15.2", "< 1.0"
```
Comment (snatchev) [inline Gemfile]: I think we also need to explicitly add `faraday_middleware` as it's a separate gem we depend on.

**Review by snatchev — APPROVED** [review]
image

👍

File: web/app/services/auth.service.spec.ts [hunk 2]
```diff
@@ -44,37 +40,32 @@ describe('AuthService', () => {
     mockLocalStorage.clear();
   });
 
-  it('should should make login request to backend', () => {
+  it('should go to github oauth UI webflow on login', () => {});
```
Comment (snatchev) [inline web/app/services/auth.service.spec.ts]: add TODO or will this complain it's empty?

**Comment by KrauseFx** [comment]
We decided to pause our work on fastlane.ci to focus on the core fastlane product as we have a lot on the roadmap. You can find more information on the [main GitHub page](https://github.com/fastlane/ci). This PR will be auto-closed 👍

**Pull request closed without merging**